docs/contributing/introduction-to-contributing.md

```diff
@@ -151,4 +151,4 @@ Do you have questions? Join the conversation in our [Discord](https://discord.gg
 ## License
 
 By contributing to the OpenSauced project, you agree that your contributions will be licensed
-under its [MIT license](https://raw.githubusercontent.com/open-sauced/open-sauced/main/LICENSE).
+under the License specified in the repo you are contributing to. This can be found in the `LICENSE` file.
```

Comment:
[Proofreading] This part could be a bit shorter. Consider using the following example:
```suggestion
by a specific License. This information can be found in the `LICENSE` file of the repo you are contributing to.
```

Reply:
That example is good, thank you!

Reply:
Thanks @dominicduffin1. All you have to do is press the Commit suggestion button.